### DISCUSSION TITLE
ruTorrent task plugin

### DISCUSSION THREAD

**Discussion by SSoft7** [Novik/ruTorrent#2387] [Q&A]
Hey all,

I'm not sure how the tasks plugin actually works. The screenshot in the wiki shows that the task plugin stores tasks of creating screenshots
https://github.com/Novik/ruTorrent/wiki/Plugin_task

But when I start creating a screenshot it does not appear in my tasks tab. The only time a task appears in the task tab is when I click the hide button while creating a torrent.

What Am I Missing here?

**Comment by stickz** [comment]
Hello @SSoft7, I just fixed an issue with the task plugin. Thanks for reporting this problem. https://github.com/Novik/ruTorrent/commit/1c80a6d4a67ea73295f17866241d261c1e34116a

I'm assuming you're running **ruTorrent v4.0.** If so, consider updating to the latest commit. You can clone it or download & extract [1c80a6d](https://github.com/Novik/ruTorrent/archive/1c80a6d4a67ea73295f17866241d261c1e34116a.zip).

**Reply by SSoft7** [reply]
Hi,

I was using [v3.10](https://github.com/Novik/ruTorrent/releases/tag/v3.10) because that is the latest release. When can we expect the proper v4 stable release listed on the releases page?
